src/client/components/VesselDetails.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import CardTeaser from "./CardTeaser";
+import { CardGroup, CardGroupHeader } from "./CardGroup";
+import VesselTeaser from "./VesselTeaser";
+
+function VesselDetailsComponent({
+  vesselName,
+  departureFrom,
+  arrivesAt,
+  etd,
+  eta,
+  date,
+  lat,
+  lon,
+  hsfo,
+  ulsfo
+}) {
+  return (
+    <div>
+      <VesselTeaser
+        vesselName={vesselName}
+        departureFrom={departureFrom}
+        arrivesAt={arrivesAt}
+        etd={etd}
+        eta={eta}
+      />
+
+      <CardGroupHeader title="Position" />
+      <CardGroup>
+        <CardTeaser title="DATE" data={date} />
+        <CardTeaser title="LATTITUDE" data={lat} />
+        <CardTeaser title="LONGITUDE" data={lon} />
+      </CardGroup>
+
+      <CardGroupHeader title="Consumption" />
+      <CardGroup>
+        <CardTeaser title="HSFO" data={hsfo} />
+        <CardTeaser title="ULSFO" data={ulsfo} />
+        <CardTeaser title="FOC TOTAL" data={hsfo + ulsfo} />
```

Comment:
Please change all of these titles to "Title Case" instead of UPPERCASE. I know it is all uppercase in the mockups, but we generally use CSS to make text like that uppercase since it is only something we do for appearance.

Imagine that the same data was used to display in another app. They might not care about it being uppercase.

Reply:
Also, I realise that since we are not doing any styling the headings will not be uppercase right now. That's intentional :)

Reply:
renamed all titles in Title Case expect HSFO,ULSFO,FOC,ETD,ETA.